docker-compose.yml

```diff
@@ -19,6 +19,7 @@ services:
     command: postgres -c 'max_connections=1000'
     ports:
       - 127.0.0.1:5432:5432
+      - 172.17.0.1:5432:5432
```

Comment:
this is the ip of docker internal host

Reply:
I haven't found any different way to expose that docker internal ip :/ And without exposing it, docker isn't able to connect to the database.

Reply:
It has an alias. 
`host.docker.internal`

you can have a look how it is done for explorer